.travis.Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM ubuntu:artful
+FROM ubuntu:18.04
 
 RUN apt-get -qq update && \
     apt-get install -y sudo docker.io git make btrfs-tools libdevmapper-dev libgpgme-dev libostree-dev
 
-ADD https://storage.googleapis.com/golang/go1.11.1.linux-amd64.tar.gz /tmp
+ADD https://storage.googleapis.com/golang/go1.11.12.linux-amd64.tar.gz /tmp
 
-RUN tar -C /usr/local -xzf /tmp/go1.11.1.linux-amd64.tar.gz && \
-    rm /tmp/go1.11.1.linux-amd64.tar.gz && \
-    ln -s /usr/local/go/bin/* /usr/local/bin/
```

Comment:
(Nitpick absolutely not worth another re-spin: removing this indentation looks unintentional.)